group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupCoordinatorConfigTest.java

```diff
@@ -46,8 +46,8 @@
 
 public class GroupCoordinatorConfigTest {
     private static final List<ConfigDef> GROUP_COORDINATOR_CONFIG_DEFS = List.of(
```

Comment:
Perhaps we should move `GROUP_COORDINATOR_CONFIG_DEFS` into `GroupCoordinatorConfig`.  `GROUP_COORDINATOR_CONFIG_DEFS` is used extensively.

Reply:
That makes sense. I will refactor this as a follow-up.